src/tinypeel/tinypeel.py

```diff
@@ -383,11 +403,12 @@ def getArgs():
             "phasefile",
             "seqfile",
             "pedigree",
+            "altalleleprobfile",
             "startsnp",
             "stopsnp",
+            "mainmetafounder"
```

Comment:
Please use `alt_allele_prob_file` and `main_meta_founder`, these two actually correspond to the user input.

Reply:
Will do. Metafounder is one word so would it be preferable to use main_metafounder?